### ISSUE TITLE
New release with WOMBAT coupling bug fix

### ISSUE THREAD

**Issue by dougiesquire** [ACCESS-NRI/ACCESS-OM2-BGC#7]
There is a bug in MOM5 in the coupling of BGC fields between the sea-ice and WOMBAT - see [here](https://github.com/COSIMA/access-om2/issues/282). This has been fixed in [this PR](https://github.com/mom-ocean/MOM5/pull/386). A few in the community are keen to test the impact of the fix.

Once the above PR is merged, we should update the ACCESS-NRI fork of MOM5 and do a new release of ACCESS-OM2-BGC that includes the fix.

**Comment by CodeGat** [comment]
Reopening this issue as we will be redoing the PR without the dependency update, as the dependency update had unintended consequences. Will link another PR that contains the above fix, that is not an update to the `OM2-BGC` dependencies

**Comment by CodeGat** [comment]
Since we have found the cause of the slowdown, we don't need a MOM5 fix without the dependency update. Instead, https://github.com/ACCESS-NRI/ACCESS-OM2-BGC/pull/14 will close this issue
